### PR TITLE
updated readme.md file

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
README.md need updating

# What has changed
README updated

# How to test?
no code changes, just read file contents

# Links
[Trello](https://trello.com/c/vag8Mo85/1305-update-readme-for-census-rm-case-api)

# Screenshots (if appropriate):

### CODE REVIEW

**Comment by AdeHarrison** [comment]
[update Readme for census-rm-case-api](https://trello.com/c/vag8Mo85/1305-update-readme-for-census-rm-case-api)

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

File: README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
 # census-rm-case-api
-This service provides a read-only API for case details.
+
+# Overview
+This case api service provides a range of Restful endpoints that -
+* Retrieve case details by case id, case ref, UPRN or QID
+* Retrieve a QID by case id
+* Create and return a new Uac Qid Link
```
Comment (LukeLoze1) [inline README.md]: Perhaps worth noting that it also does send on a rabbit msg off when creating a uac.

**Review by LukeLoze1 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (AdeHarrison) [reply README.md]: done

**Review by NickGrantX — CHANGES_REQUESTED** [review]
I know this is onerous, but with API documentation, we've often documented each endpoint with a little more detail (URL, parameters, example JSON). Not sure if this is a job for Swagger or whether you just want to cobble together something from the specs (could always link to the specs I guess).

File: README.md [hunk 2]
```diff
@@ -4,10 +4,13 @@
 This case api service provides a range of Restful endpoints that -
 * Retrieve case details by case id, case ref, UPRN or QID
 * Retrieve a QID by case id
-* Create and return a new Uac Qid Link
+* Create and return a new Uac Qid Link - this resulting
```
Comment (ryangrundy7) [inline README.md]: `- this resulting ` not meant to be there?

**Comment by gavinleeedwards** [comment]
> I know this is onerous, but with API documentation, we've often documented each endpoint with a little more detail (URL, parameters, example JSON). Not sure if this is a job for Swagger or whether you just want to cobble together something from the specs (could always link to the specs I guess).

Agree that as it's an API, it'd be nice to know the endpoints, how they're used, what they return etc.  Similar to what rm business has (https://github.com/ONSdigital/rm-case-service/blob/master/API.md)

File: README.md (hunk 2 above)
Comment (AdeHarrison) [reply README.md]: done

**Review by NickGrantX — APPROVED** [review]
Great documentation 👍

**Pull request merged**